src/components/Title.js

```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components';
 import { COLOR_CLAVE_GREEN } from '../colors';
 
 
-const StyledWrapper = styled.h1`
+const StyledWrapper = styled.h4`
```

Comment:
Bør ikke dette være en h1? Det blir ikke  riktig i hht universell utforming å bruke en h4 uten å ha h1, h2 og h3 først

Reply:
Ok, så gjøre h1 mindre da (for mobilvisning?). Teksten ble i utgangspunktet altfor stor for mobilvisning

Reply:
Ja, tror det er best å legge inn en media query for `<h1>` som setter korrekt størrelse på mobil

Reply:
Kanskje sånn?
```
@media only screen and (max-width: 600px) {
    h1 {
        font-size: 2.25rem;
    }
}
```

Reply:
fikset i: 49d938c